test/E2ETest/Microsoft.Test.E2E.AspNet.OData/Build.AspNetCore/Microsoft.Test.E2E.AspNetCore.OData.csproj

```diff
@@ -1402,30 +1402,27 @@
     <Compile Include="..\OpenType\TypedTest.cs">
       <Link>OpenType\TypedTest.cs</Link>
     </Compile>
-    <Compile Include="..\BulkOperation\BulkInsertDataModel.cs">
-      <Link>BulkOperation\BulkInsertDataModel.cs</Link>
+    <Compile Include="..\BulkOperation\BulkOperationTest.cs">
+      <Link>BulkOperation\BulkOperationTest.cs</Link>
     </Compile>
-    <Compile Include="..\BulkOperation\BulkInsertEdmModel.cs">
-      <Link>BulkOperation\BulkInsertEdmModel.cs</Link>
+    <Compile Include="..\BulkOperation\BulkOperationDataModel.cs">
+      <Link>BulkOperation\BulkOperationDataModel.cs</Link>
     </Compile>
-    <Compile Include="..\BulkOperation\BulkInsertController.cs">
-      <Link>BulkOperation\BulkInsertController.cs</Link>
+    <Compile Include="..\BulkOperation\BulkOperationEdmModel.cs">
+      <Link>BulkOperation\BulkOperationEdmModel.cs</Link>
+    </Compile>
+    <Compile Include="..\BulkOperation\BulkOperationController.cs">
+      <Link>BulkOperation\BulkOperationController.cs</Link>
     </Compile>
     <Compile Include="..\BulkOperation\BulkOperationPatchHandlers.cs">
       <Link>BulkOperation\BulkOperationPatchHandlers.cs</Link>
     </Compile>
-	<Compile Include="..\BulkOperation\BulkInsertDataModel.cs">
-		<Link>BulkOperation\BulkInsertDataModel.cs</Link>
-	</Compile>
-	<Compile Include="..\BulkOperation\BulkInsertEdmModel.cs">
-		<Link>BulkOperation\BulkInsertEdmModel.cs</Link>
-	</Compile>
-	<Compile Include="..\BulkOperation\BulkInsertController.cs">
-		<Link>BulkOperation\BulkInsertController.cs</Link>
-	</Compile>
-	<Compile Include="..\BulkOperation\BulkOperationPatchHandlers.cs">
-		<Link>BulkOperation\BulkOperationPatchHandlers.cs</Link>
-	</Compile>
+    <Compile Include="..\BulkOperation\BulkOperationDataModel.cs">
+      <Link>BulkOperation\BulkOperationDataModel.cs</Link>
+    </Compile>
+    <Compile Include="..\BulkOperation\BulkOperationEdmModel.cs">
```

Comment:
Isn't this already being done in Line 1411? https://github.com/OData/WebApi/pull/2693/files#diff-738be0b1e326687457263499cadef27a35adcd8e7a1fe79a874ce619252ab14fR1411

Reply:
Resolved